api/management/commands/index_and_notify.py

```diff
@@ -467,7 +467,10 @@ def construct_template_record(self, rtype, record):
     def notify(self, records, rtype, stype, uid=None):
         record_count = 0
         if records:
-            record_count = records.count()
+            try:
+                record_count = records.count()
+            except:
+                record_count = len(records)
```

Comment:
@batpad @frozenhelium  @thenav56 There should be an easier way to distinct if the function gets a `queryset` or an `array`?

Since if it's an `array` then `.count()` requires an argument which determines what element to count **in** the `array`.
`if 'count' in dir(records): ...`  is not a feasible option since both have the `.count()` method, just used in other ways.

Reply:
@GregoryHorvath You can use python built-in function `isinstance`
eg:
```python
from django.db.models.query import QuerySet

# For Django Queryset
isinstance(your_qs, QuerySet)

# For Array
isinstance(your_qs, list)
```